src/main/java/com/ucu/taisback/service/FirebaseService.java

```diff
@@ -96,6 +96,18 @@ public void deleteProduct(String gtin) throws InterruptedException, ExecutionExc
     firestore.collection("products").document(gtin).delete();
   }
 
+  public Product switchRedirect(String gtin) throws InterruptedException, ExecutionException, ProductNotFoundException {
+    Product product = getProduct(gtin);
+    if(product.isOnly_redirect()){
+      product.setOnly_redirect(false);
+    }
+    else{
+      product.setOnly_redirect(true);
+    }
```

Comment:
if true { true } else { false }

```suggestion
    product.setOnly_redirect(!product.isOnly_redirect());
```

Reply:
me gusta